extension/src/shared/layerContainers/hooks/useEvaluateGeneralAppearance.ts

```diff
@@ -301,10 +305,16 @@ export const useEvaluateGeneralAppearance = ({
           image:
             makeSimpleValue(pointImageValue) ?? makeConditionalImageExpression(pointImageCondition),
           imageColor: makeConditionalImageColorExpression(pointImageCondition),
-          imageSize: pointImageSize?.preset?.defaultValue ?? "__default__",
-          imageSizeInMeters: pointImageSize?.preset?.enableSizeInMeters ?? false,
```

Comment:
@airslice Are these default value necessary? I think it's unnecessary because ReEarth reset the value if specified value is undefined.

Reply:
The appearance object is also been used in override, and somehow i remember reearth override will ignore it if the value is undefined. For example in one group we override image size to 100, and when switch to another group which has no point size field component, at this moment it will override with `imageSize: undefined`, which will be ignored and the imageSize still 100. 
In the discussion early we decide to make every goup in one dataset has the same filed component to avoid this issue. But here i still added the default value since in my test i didn't follow that rule.
Anyway i want to confirm this first: should override with value undefined be ignored by reearth? I think that's the root reason of we add the cleansOverride in view 2.0.